src/index.js

```diff
@@ -0,0 +1,54 @@
+import './style.css';
+import DragAndSort from './drag.js';
+import Status from './status.js';
+import Crud from './crud.js';
+
+const list = [];
+
+function loadLiEvents() {
+  const liElements = document.querySelectorAll('.item');
+  for (let i = 0; i < liElements.length; i += 1) {
+    const drag = new DragAndSort();
+    liElements[i].addEventListener('dragstart', drag.dragStart);
+    liElements[i].addEventListener('dragover', drag.dragOver);
+    liElements[i].addEventListener('drop', drag.drop);
+  }
+  const itemDesc = document.querySelectorAll('.item-description');
+  for (let j = 0; j < liElements.length; j += 1) {
+    const crud = new Crud();
+    itemDesc[j].addEventListener('input', crud.updateTask); // Update Elements
+    itemDesc[j].addEventListener('focusin', crud.showTrashAndRemove); // Remove Elements
+    itemDesc[j].addEventListener('focusout', crud.showDots);
+  }
+}
```

Comment:
Try to don't use the old for loop in all your JS files

Reply:
I did it. 💪🏽